lib/views/settings/settings.dart

```diff
@@ -10,16 +10,14 @@ class SettingsScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: Center(
+      appBar: userType.isAdmin ? AppBar() : null,
+      body: const Center(
         child: Column(
           mainAxisSize: MainAxisSize.min,
           children: [
-            Text(
-              'Dark theme?',
-              style: TextStyle(fontSize: 20, color: context.colorScheme.onBackground),
-            ),
-            const SizedBox(height: 20),
-            const _ThemePicker(),
+            Text('Dark theme?', style: TextStyle(fontSize: 20)),
```

Comment:
The text here defaulted to `colorScheme.onSurface`, which prior to this pull request was white in light mode and very difficult to read.

Personally, I wish that Flutter had it default to `colorScheme.onBackground`, but at least it's fixed now.